.travis.yml

```diff
@@ -432,50 +432,50 @@ jobs:
       <<: *docker_amd64_linux_ubuntu_focal_rpc-tests_4
     # osx based tests
     - stage: Test
-      if: env(SKIP_OSX) != true
+      if: NOT env(SKIP_OSX) OR tag IS present
       <<: *test_osx_xcode9_4
       <<: *amd64_osx_xcode9_4_unit-tests
     - stage: Test
-      if: env(SKIP_OSX) != true
+      if: NOT env(SKIP_OSX) OR tag IS present
       <<: *test_osx_xcode9_4
       <<: *amd64_osx_xcode9_4_rpc-tests_1
     - stage: Test
-      if: env(SKIP_OSX) != true
+      if: NOT env(SKIP_OSX) OR tag IS present
       <<: *test_osx_xcode9_4
       <<: *amd64_osx_xcode9_4_rpc-tests_2
     - stage: Test
-      if: env(SKIP_OSX) != true
+      if: NOT env(SKIP_OSX) OR tag IS present
       <<: *test_osx_xcode9_4
       <<: *amd64_osx_xcode9_4_rpc-tests_3
     - stage: Test
-      if: env(SKIP_OSX) != true
+      if: NOT env(SKIP_OSX) OR tag IS present
       <<: *test_osx_xcode9_4
       <<: *amd64_osx_xcode9_4_rpc-tests_4
     - stage: Test
-      if: env(SKIP_OSX) != true
+      if: NOT env(SKIP_OSX) OR tag IS present
       <<: *test_osx_xcode9_4
       <<: *amd64_osx_xcode9_4_rpc-tests_5
     - stage: Test
-      if: env(SKIP_OSX) != true
+      if: NOT env(SKIP_OSX) OR tag IS present
       <<: *test_osx_xcode9_4
       <<: *amd64_osx_xcode9_4_rpc-tests_6
     - stage: Test
-      if: env(SKIP_OSX) != true
+      if: NOT env(SKIP_OSX) OR tag IS present
       <<: *test_osx_xcode9_4
       <<: *amd64_osx_xcode9_4_rpc-tests_7
     - stage: Test
-      if: env(SKIP_OSX) != true
+      if: NOT env(SKIP_OSX) OR tag IS present
       <<: *test_osx_xcode9_4
       <<: *amd64_osx_xcode9_4_rpc-tests_8
     - stage: Test
-      if: env(SKIP_OSX) != true
+      if: NOT env(SKIP_OSX) OR tag IS present
       <<: *test_osx_xcode9_4
       <<: *amd64_osx_xcode9_4_rpc-tests_9
     - stage: Test
-      if: env(SKIP_OSX) != true
+      if: NOT env(SKIP_OSX) OR tag IS present
       <<: *test_osx_xcode9_4
       <<: *amd64_osx_xcode9_4_rpc-tests_10
     - stage: Test
-      if: env(SKIP_OSX) != true
+      if: NOT env(SKIP_OSX) OR tag IS present
```

Comment:
All of these `if: NOT env(SKIP_OSX) OR tag IS present` lines should not be necessary, as the `<<: *test_osx_xcode9_4` yml anchor pulls in line 297 `if: (NOT env(SKIP_TESTS)) AND (NOT env(SKIP_OSX)) OR tag IS present # Never skip MacOS when running the CI for a tag`

Reply:
I've just removed the redundant checks as you suggested, thanks!